odoo/api.py

```diff
@@ -948,6 +948,15 @@ def contains(self, record, field):
         """ Return whether ``record`` has a value for ``field``. """
         return record.id in self._get_field_cache(record, field)
 
+    def contains_field(self, field):
+        """ Return whether ``field`` has a value for at least one record. """
+        cache = self._data.get(field)
+        if not cache:
+            return False
+        if isinstance(next(iter(cache)), tuple):  # For context dependent field
```

Comment:
```suggestion
        # 'cache' keys are tuples if 'field' is context-dependent, record ids otherwise
        if isinstance(next(iter(cache)), tuple):
```